temporian/implementation/numpy/operators/test/unique_timestamps_test.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from absl.testing import absltest
+
+import pandas as pd
+from temporian.core.operators.unique_timestamps import UniqueTimestamps
+from temporian.implementation.numpy.data.event import NumpyEvent
+from temporian.implementation.numpy.operators.unique_timestamps import (
+    UniqueTimestampsNumpyImplementation,
+)
+
+
+class UniqueTimestampsOperatorTest(absltest.TestCase):
+    def setUp(self):
+        pass
+
+    def test_base(self):
+        event_data = NumpyEvent.from_dataframe(
+            pd.DataFrame(
+                {
+                    "timestamp": [1, 2, 2, 2, 3, 3, 4],
```

Comment:
can we add a `3` to `c` index key `1` (so that we get multiple `3` timestamps, one for `c=1` and one for `c=2`)

Reply:
*Done